rosetta-client/src/client.rs

```diff
@@ -47,6 +47,10 @@ impl GenericClient {
                 let client = EthereumClient::new("polygon", network, url).await?;
                 Self::Ethereum(client)
             },
+            Blockchain::Arbitrum => {
+                let client = EthereumClient::new("Arbitrum", network, url).await?;
```

Comment:
This should be in lower case, otherwise `std::str::FromStr` doesn't work.
```suggestion
                let client = EthereumClient::new("arbitrum", network, url).await?;
```